gradle.properties

```diff
@@ -24,4 +24,4 @@ kotlin.daemon.jvmargs=-Xmx2g -Xms500m
 nodeBinaries.commit=8755ae4c05fd476cd23f2972049111ba436c86d4
 nodeBinaries.version=v20.12.2
 cody.autocomplete.enableFormatting=true
-cody.commit=daa1693620fdf2784ecd190dd12bb07169950a90
+cody.commit=5bd253c16261680490d2afb14398ff568334ffbd
```

Comment:
why do we need to update the cody commit?

Reply:
It is not needed.
I had some problems which I thought were related to bugs in cody main but it turned out to not be the case.
I reverted this change for now.

But we should update it. I will do it in a separate PR.